genai-perf/README.md

```diff
@@ -105,9 +105,8 @@ You can also build Perf Analyzer [from source](../docs/install.md#build-from-sou
 ### Install GenAI-Perf from source
 
 ```bash
-git clone https://github.com/triton-inference-server/perf_analyzer.git && cd perf_analyzer
-
-pip install -e genai-perf
+pip install git+https://github.com/triton-inference-server/perf_analyzer.git#subdirectory=gen
+ai-perf
```

Comment:
nit: place on one line or add `\` line break or something, currently triple clicking only grabs first line, not both. (I know there's also a copy button on github rendered version, but applies generally when rendered to docs website, or cat'd from terminal, etc
```suggestion
pip install git+https://github.com/triton-inference-server/perf_analyzer.git#subdirectory=genai-perf
```

Reply:
That had to have been the pre-commit formatter haha. I think it exceeds the character limit for one line. I added `\` and put the `git+...` on a second line.

Reply:
nvm. moved all to one line. idk why it was ever split like that